README.md

```diff
@@ -14,13 +14,13 @@ This book contains some common **DOs** & **DON'Ts** for Open Source software.
 
 The Open Source community is thriving. Each day the number of Open Source projects grows, as does the army of contributors that maintain them. While this is exciting for the industry, it can be daunting as a developer new to the community. This book aims to provide some tips for newcomers to help them avoid the pitfalls of Open Source development and learn from the community's collective wisdom.
 
-As the ancient proverb goes, _Time and tide [and technology] wait for no man_. And to the best of our ability, neither will this book. Remember to check the version number for updates!
+As the ancient proverb goes, Time and tide [and technology] wait for no man. And to the best of our ability, neither will this book. Remember to check the version number for updates!
```

Comment:
The `_` is for _italic text_, so is still needed. If you click `view` at the top right of the file on GitHub, it will show you the rendered version